helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml

```diff
@@ -236,12 +236,13 @@ spec:
     # Kyverno-related resource counts by kind
     - expr: sum(etcd_kubernetes_resources_count{kind=~".*.kyverno.io|clusterpolicyreports.wgpolicyk8s.io|policyreports.wgpolicyk8s.io"}) by (cluster_id, kind)
       record: aggregation:kyverno_resource_counts
-    # Kyverno policy status by team
+    # Kyverno policy status by team - Deployments
```

Comment:
Maybe @hervenicol or @TheoBrigitte have a better solution in mind.

I would be careful about noteam though. We might want to add a metric in grafana cloud to know if we have apps without teams

Reply:
Yes I meant that for my team :)